bokeh/models/glyph.py

```diff
@@ -55,6 +57,8 @@ class Glyph(Model):
 
     '''
 
+    decorations = List(Instance(Decoration), default=[])
```

Comment:
Also is this not too general? This would admit any decoration on any glyph and that seems like it will not always result in sensible combinations?

Reply:
This is possibly too general, but we will see as new decoration variants and other related features are implemented. This will definitely be refined before 3.0.